utils/local-setup.sh

```diff
@@ -61,8 +61,12 @@ set -e pipefail
 trap cleanup EXIT 1 2 3 6 15
 
 cleanup() {
-  echo "Killing KCP"
-  kill "$KCP_PID"
+  if [[ -n ${LOCAL_SETUP_BACKGROUND} ]] then
+    echo "Exiting. Use 'pkill kcp' to shutdown kcp"
```

Comment:
```suggestion
  if [[ -n ${LOCAL_SETUP_BACKGROUND} ]]
  then
    echo "Exiting. Use 'pkill kcp' to shutdown kcp"
```
This gave an error for me locally `./utils/local-setup.sh: line 64: syntax error near unexpected token `then'`

Reply:
right, I don't know how it didn't fail for me